libcontainer/container_linux.go

```diff
@@ -35,7 +35,7 @@ import (
 
 const stdioFdCount = 3
 
-type linuxContainer struct {
+type Container struct {
```

Comment:
Perhaps copy the previous docs here?

```suggestion
// Container is a libcontainer container object.
//
// Each container is thread-safe within the same process. Since a container can
// be destroyed by a separate process, any function may return that the container
// was not found.
type Container struct {
```

Reply:
Frankly, I do not understand the part about thread-safeness, and the rest is kind of obvious.